### ISSUE TITLE
Null Pointer iOS Google Maps Extra Parameters

### ISSUE THREAD

**Issue by fripas** [dpa99c/phonegap-launch-navigator#212]
**I'm submitting a ...**  (check one with "x"):
- [ x ] bug report
- [ ] feature request
- [ ] documentation issue

# Bug report

Null pointer extra parameters

**Current behavior:**

When you try to open Google Maps and do not send extra parameters, it does not open anything.

**Expected behavior:**

Google Maps App open with parameters

**Steps to reproduce:**

Ionic 4 + @ionic-native

Output Console:
`unrecognized selector sent to instance 0x1b5984878`

**Environment information**

- Cordova CLI version 
	8.1.2 (cordova-lib@8.1.1)
- Cordova platform version
	Installed platforms:
           android 7.1.4
           ios 4.5.5
- Plugins & versions installed in project (including this plugin)
    cordova-plugin-actionsheet 2.3.3 "ActionSheet"
    cordova-plugin-device 2.0.2 "Device"
    cordova-plugin-dialogs 2.0.1 "Notification"
    cordova-plugin-googlemaps 2.5.0 "cordova-plugin-googlemaps"
    cordova-plugin-ionic-keyboard 2.1.3 "cordova-plugin-ionic-keyboard"
    cordova-plugin-ionic-webview 2.3.2 "cordova-plugin-ionic-webview"
    cordova-plugin-mauron85-background-geolocation 3.0.0-alpha.50 "CDVBackgroundGeolocation"
    cordova-plugin-splashscreen 5.0.2 "Splashscreen"
    cordova-plugin-statusbar 2.4.2 "StatusBar"
    cordova-plugin-whitelist 1.3.3 "Whitelist"
    uk.co.workingedge.phonegap.plugin.launchnavigator 5.0.0 "Launch Navigator"
- Dev machine OS and version, e.g.
    - OSX
        ProductName:	Mac OS X
        ProductVersion:	10.13.6
        BuildVersion:	17G65
        
_Runtime issue_
- Device details
    - iPhone 6S
- OS details
    - iOS 11.4.1
	
_iOS build issue:_
- Node JS version
    v10.15.0
- XCode version
    Version 10.1 (10B61)
_If using an [Ionic Native Typescript wrapper]() for this plugin:_
- Ionic environment info
    
Ionic:

   ionic (Ionic CLI)             : 4.10.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.0
   @angular-devkit/build-angular : 0.12.3
   @angular-devkit/schematics    : 7.2.3
   @angular/cli                  : 7.2.3
   @ionic/angular-toolkit        : 1.2.3

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4, browser 5.0.4, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.3.2, (and 7 other plugins)

System:

   ios-deploy : 1.9.2
   ios-sim    : 5.1.0
   NodeJS     : v10.15.0 (/usr/local/bin/node)
   npm        : 6.4.1
   OS         : macOS High Sierra
   Xcode      : Xcode 10.1 Build version 10B61

- Installed Ionic Native modules and versions
├─┬ @ionic-native/background-geolocation@5.0.0
├─┬ UNMET PEER DEPENDENCY @ionic-native/core@5.0.0
├─┬ @ionic-native/google-maps@5.0.0-beta.26
├─┬ @ionic-native/launch-navigator@5.0.0
├─┬ @ionic-native/splash-screen@5.0.0
├─┬ @ionic-native/status-bar@5.0.0

**Related code:**
```

    let options: LaunchNavigatorOptions = {
      start: [this.datos.usuario.ubicacion.lat, this.datos.usuario.ubicacion.lng],
      app: this.launchNavigator.APP.GOOGLE_MAPS,
      enableGeocoding: false
    };
    
    this.launchNavigator.navigate([this.datos.destino.lat, this.datos.destino.lng], options)
      .then(
        success => console.log('Launched navigator'),
        error => console.log('Error launching navigator', error)
      );

```

**Console output**

<details>
<summary>console output</summary>

`unrecognized selector sent to instance 0x1b5984878`

</details><br/><br/>

**Other information:**

**FIX:** 
  
**File**: _src/ios/lib/LN_LaunchNavigator.m_
`Line`: ~371
`Details`:
```
if(extras){
        [url appendFormat:@"%@", [self extrasToQueryParams:extras]];
}
```

`Change for`:
```
if(![self isNull:navigateParams[@"extras"]] && extras){
        [url appendFormat:@"%@", [self extrasToQueryParams:extras]];
}
```

**Comment by dpa99c** [comment]
Fix published to npm in v5.0.1

**Comment by tavishaggarwal** [comment]
I am also facing the same issue. And as per the issue, the fix is published to version 5.0.1.
When I run command ionic cordova plugin add uk.co.workingedge.phonegap.plugin.launchnavigator@5.0.1 it's still installing 5.0.0. How can I install 5.0.1 version of the plugin?
